### PR TITLE
Remove nextest filtration

### PR DESCRIPTION
**Pull request by AndreiEres**
Fixes https://github.com/paritytech/polkadot-sdk/issues/3884#issuecomment-2026058687

After moving regression tests to benchmarks (https://github.com/paritytech/polkadot-sdk/pull/3741) we don't need to filter tests anymore.

### CODE REVIEW

**Comment by sandreim** [comment]
Thanks @AndreiEres . It it fixes the issue CI should fail and my collation generation fix needs to be merged first in order for this PR to pass.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 3/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5691855

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 3/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5693415

**Review by alexggh — APPROVED** [review]

**Review by lexnv — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 1/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5697495

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 1/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5711989

**Review by bkchr — APPROVED** [review]

**Review by rcny — APPROVED** [review]

**Comment by sandreim** [comment]
> The CI pipeline was cancelled due to failure one of the required jobs. **Job name**: test-linux-stable 1/3 **Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5722799

Weird, locally all tests pass, merging master once again....

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 1/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5729209

File: substrate/frame/scheduler/src/tests.rs [hunk 1]
```diff
@@ -1501,8 +1501,11 @@ fn scheduler_handles_periodic_unavailable_preimage() {
 		run_to_block(4);
 		assert_eq!(logger::log().len(), 1);
 
-		// Unnote the preimage
+		// As the public api doesn't support to remove a noted preimage, we need to first unnote it
```
Comment (ggwpez) [inline substrate/frame/scheduler/src/tests.rs:1504]: @bkchr the admin origin can unnote it here. Which is why i added that `if is_requested...` before.  
Doing this still causes an underflow `assert_ok!(Preimage::unnote_preimage(RuntimeOrigin::root(), hash));`.

Comment (bkchr) [reply substrate/frame/scheduler/src/tests.rs:1504]: If the admin origin does this, it is a manual intervention. However, the code does not really underflows as the logic checks that the `count` is bigger than `1` etc. 

I don't want to modify production code to make it work in the edge case for a test.
